### PR TITLE
CP-731 Add authors, description, and homepage to pubspec.yaml

### PR DESCRIPTION
**Pull request by evanweible-wf**
## Issue
- Pubspec.yaml is missing some required properties for publishing

## Changes
- Add description, authors, and homepage to pubspec.yaml.

## Areas of Regression
- n/a

## Testing
- CI build passes.

## Code Review
@trentgrover-wf
@maxwellpeterson-wf

### CODE REVIEW

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by trentgrover-wf** [comment]
+1

**Comment by trentgrover-wf** [comment]
@jayudey-wf ready for merge

File: pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,13 @@
 name: state_machine
 version: 1.0.0
+description: Finite state machine library. Easily define legal state transitions. Listen to state entrances, departures, and transitions.
+authors:
+  - Workiva Client Platform Team <clientplatform@workiva.com>
+  - Evan Weible <evan.weible@workiva.com>
+  - Max Peterson <maxwell.peterson@workiva.com>
+  - Trent Grover <trent.grover@workiva.com>
+homepage: https://github.com/Workiva/w_service
```
Comment (trentgrover-wf) [inline pubspec.yaml]: `w_service`?

**Comment by evanweible-wf** [comment]
@trentgrover-wf @maxwellpeterson-wf fixed

**Comment by trentgrover-wf** [comment]
+1

**Comment by maxwellpeterson-wf** [comment]
+1
@jayudey-wf ready for merge

**Comment by jayudey-wf** [comment]
QA Resource Approval: +10

- [ ] Testing instruction
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
 - passing CI
- [ ] Unit test created/updated
- [x] All unit tests pass

Merging into master.

**Pull request merged**